pylintrc

```diff
@@ -47,7 +47,9 @@ load-plugins=
 # D102 Missing docstring in public method
 # D105 Missing docstring in magic method
 disable=C0111,D101,D102,D105,too-few-public-methods,too-many-instance-attributes,too-many-arguments,
- protected-access,invalid-name,redefined-outer-name,fixme
+ protected-access,invalid-name,redefined-outer-name,no-self-use,too-many-public-methods,too-many-lines,
+ no-member,access-member-before-definition,attribute-defined-outside-init,duplicate-code,fixme
```

Comment:
We might be able to re-add some of these, but it'll require changing the structure of the code.
The dynamic nature of `model.py` confuses the linter.